mlir/test/Integration/Dialect/Vector/CPU/ArmSVE/test-deinterleave.mlir

```diff
@@ -0,0 +1,18 @@
+// RUN: mlir-opt %s -test-lower-to-llvm | \
```

Comment:
Could you rename this file `test-scalable-deinterleave.mlir` (to be consistent with `test-scalable-interleave.mlir`.

Reply:
And remove "test" from the name - it's clear that it's a _test_ file without the extra letters :)

Reply:
All the vector integration tests are named like `test-<operation>.mlir`. If we want to change that we should change all of them (rather than be inconsistent).

Reply:
This is to be implemented until this patch https://github.com/llvm/llvm-project/pull/93521 is approved.